pbsmetrics/prometheus/prometheus.go

```diff
@@ -187,6 +188,12 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 		"Count of total requests to Prebid Server labeled by account.",
 		[]string{accountLabel})
 
+	metrics.requestsQueueTimer = newHistogram(cfg, metrics.Registry,
+		"request_queue_time",
+		"Seconds request was waiting in queue",
+		[]string{requestTypeLabel, requestStatusLabel},
+		prometheus.DefBuckets)
```

Comment:
You don't want to use DefBuckets. That's geared toward network connectivity measurements and maxes out at 5 seconds. Please use a new custom bucket series. I think you're expecting wait times up 5 minutes? Consider then buckets like:

0, 1, 5, 30, 60, 120, 180, 240, 300

The series (which I just guessed at) is geared towards measuring fast responses with less resolution spread equally up to 5 minutes.  

Note: Influx records time series differently and doesn't have this concern.

Reply:
Makes sense, modified

Reply:
Great. Thank you.